source/structured-streaming.txt

```diff
@@ -307,6 +428,50 @@ Stream to MongoDB from a CSV File
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To create a :ref:`write stream <write-structured-stream>` to 
+         MongoDB from a ``.csv`` file, first create a `DataStreamReader <https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamReader.html>`__ 
+         from the ``.csv`` file, then use that ``DataStreamReader`` to 
+         create a `DataStreamWriter <https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamWriter.html>`__ 
+         to MongoDB. Finally, use the ``start()`` method to begin the 
+         stream.
+         
+         As streaming data is read from the ``.csv`` file, it is added 
+         to MongoDB in the `outputMode <https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamWriter.html#outputMode-java.lang.String->`__ 
+         you specify.
+
+         .. code-block:: java
+            :copyable: true
+
+            // create a local SparkSession
+            SparkSession spark = SparkSession
+              .builder()
```

Comment:
Suggestion: Similar to the other builders (see line 457: "spark.readStream()"), call it on the same line rather than a new one.

Applies to all instances.